src/ContactPage/Contact.scss

```diff
@@ -0,0 +1,68 @@
+@import "../styles/typography";
+
+.contact-section {
+  height: 100vh;
+
+  &__border {
+    border: none;
+    padding: 2% 0;
+    background: rgb(229, 226, 226);
+  }
+  &__title {
+    font-family: $base-font-family;
+    margin: 0 5%;
```

Comment:
This and the form have the same margin, why not put the margin on the parent, _border_, element?